lib/modules/dosomething/dosomething_signup/includes/Signup.php

```diff
@@ -88,6 +88,18 @@ public static function find(array $filters = []) {
   private function build($data) {
     $this->id = $data->sid;
     $this->created_at = $data->timestamp;
+
+    $user = user_load($data->uid);
+
+    $this->user = [
+      'drupal_id' => $data->uid,
+      'northstar_id' => dosomething_helpers_isset($data->field_northstar_id_value, 'id'),
+      'first_name' => dosomething_helpers_isset(dosomething_helpers_extract_field_data($user->field_first_name), 'first_name'),
+      'last_initial' => dosomething_helpers_isset(dosomething_helpers_extract_field_data($user->field_last_name), 'last_initial'),
+      'photo' => dosomething_helpers_isset(dosomething_helpers_extract_field_data($user->photo), 'photo'),
+      'country' => dosomething_helpers_isset($user->field_address[LANGUAGE_NONE][0]['country'], 'country'),
```

Comment:
Also just to note here, not sure if it's an issue or not, but we're grabbing `user` object data from DS here, but elsewhere in the other endpoints where user data is populated, it's always from NS. I know the point of this update is to make the NS id available without doing more calls to NS. Thoughts on maybe just including the NS id and saying null to everything thing else otherwise? Alternatively, a future update could be a tiny UserResolver class or something that can check if NS has the data and if not then check Drupal and if not, then `null`. Maybe it allows passing a boolean so that we tell it specifically to _not_ check NS if we want to keep the search localized to just in Drupal too... just some ramblings and thoughts.